File: scala-pr/54/scala/code-quality-naming/ItemManager.scala
import scala.collection.mutable

class ItemManager {
  // {fact rule=code-quality-naming@v1.0 defects=1}
  private var stuff: Option[mutable.Map[String, String]] = None

  def doThing(thing1: String, thing2: String): Boolean = {
    if (stuff.isEmpty) {
      stuff = Some(mutable.Map.empty[String, String])
    }

    val temp = thing1 + thing2
    stuff.get(thing1) = temp
    
    stuff.get.contains(thing1)
  }

  def getThing(key: String): Option[String] = {
    stuff.flatMap(_.get(key))
  }
  // {/fact}
}

File: kotlin-pr/50/kotlin/code-quality-naming/ItemManager.kt
class ItemManager {
    // {fact rule=code-quality-naming@v1.0 defects=1}
    private var stuff: MutableMap<String, String>? = null

    fun doThing(thing1: String, thing2: String): Boolean {
        if (stuff == null) {
            stuff = mutableMapOf()
        }

        val temp = thing1 + thing2
        stuff!![thing1] = temp
        
        return stuff!!.containsKey(thing1)
    }

    fun getThing(key: String): String? {
        return stuff?.get(key)
    }
    // {/fact}
}